src/data/builder-tools.js

```diff
@@ -1020,4 +1020,4 @@ function ensureShowcaseValid(showcase) {
   }
 }
 
-Showcases.forEach(ensureShowcaseValid);
+Showcases.forEach(ensureShowcaseValid);
```

Comment:
I just compared it to the current version in your branch https://github.com/cardano-foundation/developer-portal/blob/staging/src/data/builder-tools.js and it is exactly the same. Does it only show up as different because of the previous commits?

Reply:
It shows up at the same because whatever program you are using to compare them is ignoring the newline at the end of the file.

If in doubt about how to fix this, you can download a fresh copy of `src/data/builder-tools.js` in the `staging` branch and then do another commit and push.  Then you will see the message about having modified that file... by creating a slightly different white space syntax... disappear from this page that shows all the diffs: https://github.com/cardano-foundation/developer-portal/pull/865/files

This is a lot of work over a single character at the end of the file for both of us, but given the previous discussion I believe the commit history should _not_ show changes for a file that we've agreed above not to modify.

Reply:
I am really sorry it took so long, the last time I copied the original document text and saved it. Git did not show any changes, therefore I could not commit and thought they were the same. Now replacing the file directly git noticed the difference. 

I hoped it is ok now